### PR TITLE
Document Templates proxy

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by ohadlevy** [comment]
@ShimShtein  can you review please?

**Comment by mmoll** [comment]
closes #289?

File: _includes/manuals/1.7/4.3.13_smartproxy_templates.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="alert alert-error">
+The templates module requires HTTP to be enabled. It is reccomended to configure all other modules to exclusively listen on HTTPS by changing <code>:enabled:</code> to <code>https</code> in their respective configurations.
```
Comment (domcleal) [inline _includes/manuals/1.7/4.3.13_smartproxy_templates.md:2]: 'https' isn't valid in 1.7

**Comment by stbenjam** [comment]
Ah, nevermind then - will reopen against 1.8 when it's branched

**Pull request closed without merging**